Domain/Sia.Domain/Event.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.Dynamic;
 using System.Text;
+using System.Linq;
```

Comment:
this `using` doesn't seem to be related to the change below.

Reply:
you are right, this was for previous version of code. Removing it.